Dockerfile

```diff
@@ -5,6 +5,9 @@ VOLUME /usr/share/nginx/html
 VOLUME /etc/nginx
 
 COPY index.html /usr/share/nginx/html/index.html
+COPY index.css /usr/share/nginx/html/__assets/index.css
+COPY github-icon.svg /usr/share/nginx/html/__assets/github-icon.svg
+COPY civic-logo-animated.svg /usr/share/nginx/html/__assets/civic-logo-animated.svg
 COPY strapdown.js /usr/share/nginx/html/__assets/strapdown.js
 COPY strapdown.css /usr/share/nginx/html/__assets/strapdown.css
```

Comment:
Since you removed strapdown from the html page can you also remove it from here?